x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.tsx

```diff
@@ -164,6 +164,7 @@ const AlertContextMenuComponent: React.FC<AlertContextMenuProps & PropsFromRedux
   });
   const { eventFilterActionItems } = useEventFilterAction({
     onAddEventFilterClick: handleOnAddEventFilterClick,
+    disabled: !isEvent,
```

Comment:
also will need to change this to `!isEvent && !isAlert` and add something like

```
  const isAlert = useMemo(
    () =>
      ['signal', 'alert'].some((eventKind) => indexOf(ecsRowData.event?.kind, eventKind) !== -1),
    [ecsRowData]
  );
```

@madirey - Question. The `event.kind` shows up as `signal` on the alert page and as `alert` on the hosts page. Is that expected or something we should look into? I think it might be associated with the rule changes

Reply:
So this is what we want, make this option available only for `events` and not for others like `alerts`